core/core_integrations/facebook.js

```diff
@@ -1,170 +1,170 @@
+                        getSenderInfo(info.participantIDs, api, event, finished);
+                });
+        }
+        else {
+                getSenderInfo([event.senderID], api, event, finished);
+        }
 };
 
 var stopTyping = function() {
-	if (endTyping) {
-		endTyping();
-		endTyping = null;
-	}
+        if (endTyping) {
+                endTyping();
+                endTyping = null;
+        }
 };
 
 exports.start = function(callback) {
-	fb({email: this.config.username, password: this.config.password}, function (err, api) {
-		if(err) {
-			console.error(err);
-			process.exit(-1);
-		}
+        fb({email: this.config.username, password: this.config.password}, function (err, api) {
+                if(err) {
+                        console.error(err);
+                        process.exit(-1);
+                }
 
-		var options = {
-			listenEvents: true
-		};
-		if (!console.isDebug()) {
-			options.logLevel = 'silent';
-		}
-		api.setOptions(options);
-		platformApi = api;
+                var options = {
+                        listenEvents: true
+                };
+                if (!console.isDebug()) {
+                        options.logLevel = 'silent';
+                }
+                api.setOptions(options);
+                platformApi = api;
 
-		platform = shim.createPlatformModule({
-			commandPrefix: exports.config.commandPrefix,
-			sendMessage: function(message, thread) {
-				stopTyping();
-				api.sendMessage({body:message}, thread);
-			},
-			sendUrl: function(url, thread) {
-				stopTyping();
-				api.sendMessage({body: url, url: url}, thread);
-			},
-			sendImage: function(type, image, description, thread) {
-				stopTyping();
-				switch (type) {
-					case "url":
-						api.sendMessage({body: description, url: image}, thread, function(err, messageInfo) {
-							if (err) {
-								api.sendMessage(description + " " + image, thread);
-							}
-						});
-						break;
-					case "file":
-						api.sendMessage({body: description, attachment: fs.createReadStream(image)}, thread);
-						break;
-					default:
-						api.sendMessage(description, thread);
-						api.sendMessage(image, thread);
-						break;
-				}
-			},
-			sendFile: this.sendImage,
-			sendTyping: function(thread) {
-				stopTyping();
-				api.sendTypingIndicator(thread, function(err, end) {
-					if (!err) {
-						endTyping = end;
-					}
-				});
-			},
-			setTitle: function(title, thread) {
-				stopTyping();
-				api.setTitle(title, thread);
-			},
-			getUsers: function(thread) {
-				return threadInfo[thread];
-			}
-		});
+                platform = shim.createPlatformModule({
+                        commandPrefix: exports.config.commandPrefix,
+                        sendMessage: function(message, thread) {
+                                stopTyping();
+                                api.sendMessage({body:message}, thread);
+                        },
+                        sendUrl: function(url, thread) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 20 space characters but found 24.](https://www.codacy.com/app/theprojectshop/Kassy/file/2555194706/issues/source?bid=3282987&fileBranchId=3291254#l75)